pallets/ddc-customers/src/lib.rs

```diff
@@ -258,7 +261,7 @@ pub mod pallet {
 				}
```

Comment:
check is already removed

Reply:
It doesn't make sense to me. Let's discuss on Monday

Reply:
Discussed in Slack. We don't need to check the genesis build.